.goreleaser.yml

```diff
@@ -28,6 +28,26 @@ brews:
     test: |
       system "#{bin}/dagger version"
 
+# https://goreleaser.com/customization/nix/
+nix:
+  - name: dagger
+    repository:
+      owner: dagger
+      name: nix
```

Comment:
There is a token option that allows setting a different token from the one pushing to this GitHub repository. Not sure if it's needed, but I guess not, because Goreleaser can push to the Homebrew repo as well. (Just thought I would mention it)

Also, Goreleaser can open a PR instead of pushing to the default branch directly if that's what you prefer. (I think it's better if it pushes directly, but I'll leave that decision to you)

See details here: https://goreleaser.com/customization/nix/

Reply:
It's OK if GoReleases pushes to this branch directly. We do the same with https://github.com/dagger/homebrew-tap